samples/nrf9160/gnss/src/main.c

```diff
@@ -226,6 +237,16 @@ static int modem_init(void)
 	}
 
 	LOG_INF("Connected to LTE network");
+
+	if (IS_ENABLED(CONFIG_DATE_TIME)) {
+		/* Wait for an event from the Date Time library. */
```

Comment:
There could be a print here saying that we're waiting on valid time to be obtained, just in case a while and users wonder why the sample doesn't proceed

Reply:
True, added.